### PR TITLE
Nagios support for monitor:health

### PR DESCRIPTION
**Pull request by dav-m85**
Add --nagios option to the monitor:health command, making it suitable to use with NRPE plugins.

```--nagios``` implies RawConsoleReporter (usual reporter without the coloring), and returnCodes following the NRPE norm (0: ok, 1: warning, 2:critical).

### CODE REVIEW

**Comment by dav-m85** [comment]
I just noticed that rawConsoleReporter could be avoided by forcing ```--no-ansi``` option. I will remove it.

File: Command/HealthCheckCommand.php [hunk 1]
```diff
@@ -36,13 +43,34 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+            } else if($results->getWarningCount()) {
+                $returnCode = 1;
+            } else {
+                $returnCode = 0; // We may have som skipped although
+            }
+        } else {
+            $returnCode = $results->getFailureCount() ? 1 : 0;
+        }
+
+        return $returnCode;
```
Comment (lsmith77) [inline Command/HealthCheckCommand.php]: I would just return immediately on all these places. its always good to return early and this way we do not need the else either

**Comment by lsmith77** [comment]
let me know when you had time to do those clean ups. thanks!

**Comment by lsmith77** [comment]
ping?

**Comment by dav-m85** [comment]
this is it :)

**Comment by lsmith77** [comment]
almost. can you remove the two else statements?

**Comment by dav-m85** [comment]
You mean like this ?

**Comment by lsmith77** [comment]
great

**Pull request merged**